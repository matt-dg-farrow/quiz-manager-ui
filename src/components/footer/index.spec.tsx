import { shallow } from "enzyme";
import toJson from "enzyme-to-json";
import React from "react";
import Footer from ".";

describe("The Header component", () => {
  const wrapper = shallow(<Footer />);

  it("should contain the correct number of components", () => {
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
