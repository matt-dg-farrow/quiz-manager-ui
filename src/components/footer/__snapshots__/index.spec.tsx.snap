// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The Header component should contain the correct number of components 1`] = `
<footer
  style={
    Object {
      "boxShadow": "none",
    }
  }
>
  <div
    className="wrapper-footer"
  >
    <div>
      <p
        className="footer-message"
      >
        WebbiSkools Ltd
      </p>
    </div>
    <div
      className="footer-menu"
    >
      <ul>
        <li>
          <a
            href="#0"
          >
            Feedback
          </a>
        </li>
        <li>
          <a
            href="#0"
          >
            Help
          </a>
        </li>
      </ul>
    </div>
  </div>
</footer>
`;
